.circleci/config.yml

```diff
@@ -736,3 +727,21 @@ jobs:
           path: ~/project/results
       - store_test_results:
           path: ~/project/results
+
+  share-testing-image:
+    machine:
+      image: ubuntu-2004:202010-01
+    working_directory: /home/circleci/go/src/github.com/influxdata/influxdb
+    steps:
+      - attach_workspace:
+          at: /home/circleci/go/src/github.com/influxdata/influxdb
+      - run:
+          name: Log into Quay
+          command: docker login -u "$QUAY_USER" -p $QUAY_PASS quay.io
+      - run:
+          name: Push the image to Quay
+          command: |
+            docker load < docker-image.tar
```

Comment:
Following up from comments on `build`: The `docker build` steps could run here. IMO the `e2e` job could continue to test the un-dockerized binary for as long as we keep it around.

Reply:
So running the `e2e` job with the docker image is largely ceremonial and actually caused some additional complications described elsewhere, but I was thinking it would be good to do it this way to have some confidence that the docker image actually works with the UI before unleashing it into the world and having the `ui` repo run its CI against it. Fundamentally if the binary works then the docker image should work too...so maybe I'm being a little too paranoid.

Reply:
Makes sense to me, I guess it did flush out some problems like the video and flaky test. Good to leave dockerized now that it's working IMO 👍